Add 2022-03-15 minutes (#28)
opencontainers/wg-reference-types: minutes/2022-03-15.md

    - <https://github.com/opencontainers/wg-reference-types/pull/24>
    - <https://github.com/opencontainers/wg-reference-types/pull/23>
- Proposal C (Nisha?)
    - <https://github.com/opencontainers/wg-reference-types/blob/main/docs/proposals/PROPOSAL_C.md>
- Reference Looping (Brandon)
- _enter agenda before meeting_

Notes
- Should we update the template to include discussion and feedback (Brandon). 
   - Should we add the rubric and feedback to the template? (Brandon)
   - The intent of the template was to not need to much review and am concerned (Josh) 
   - If people are challenging a yes/no to the rubric - where should we have that discussion (Nisha)
   - I would like to the proposal to be the one stop shop (Brandon)
   - I thought of the proposal docs to be a snapshot and not to be continuously updated (Josh)
   - I don't think bloating the proposal document is the right
   - DECISION to punt the feedback decision until later and track the comments on the proposal addition to the rubric PR
- Proposal C discussion
   -  Introduces a new node manifest
   -  node manifest looks very similar to index however it adds a description and a manifest blob
   -  Questions

>    -  When we're looking at objects field, are they manifest or objects or both and how are they accessed and interact? (Brandan)

Reviewer: ```suggestion
   -  When we're looking at objects field, are they manifest or blobs or both and how are they accessed and interact? (Brandon)
```

   -  If object can point to a blob or other objects it gets confusing for the client to understand which API to send the request to
   -  If you're writing a client that's trying to construct the tree, I don't know how to do it.
   -  If I push a v2 image manifest, do I need to push a v3 at the same time or is it created and I could. Can I point to v2 manifests from v3 APIs? (Jason)
   -  Confusion about the addition of the objects? Can you help explain when they are used (Josh)
   -  Can you walk through the series of pushing to understand the process. What is the entrypoint? (Josh) The reference field is still causing confusing
   -  Is this similar to what we tried to do in oras artifacts? Ref: https://github.com/oras-project/artifacts-spec/pull/75 (Steve)
   -  How do you recommend we would filter for objects of a specific type? (Sajay)
       -  Artifact type has been specifically left out because there were use cases where IANA types were not favored
   -  What are you thoughts on backwards compatibility? (Brandon)

>    -  If I copy this a registry th at supports v3 to a registry that supports v2? (Brandon)

Reviewer: ```suggestion
   -  If I copy this from a registry that supports v3 to a registry that supports v2? (Brandon)
```

Author: Thanks Brandon - Will fix
